installer/functions/plugin/helper-plugins.private.ts

```diff
@@ -0,0 +1,163 @@
+
+import axios from 'axios';
+import { ApplicationContext, Configuration } from '../../src/constants/interface';
+import { 
+  ACCOUNT_SID,
+  AUTH_TOKEN,
+  getParam
+} 
+from '../common';
+
+export const listPluginsEndpoint = "https://flex-api.twilio.com/v1/PluginService/Plugins";
+
+/**
+ * Needed to make API calls
+ * @param stringToConvert 
+ * @returns - base64 string
+ */
+export const convertToBase64 = (stringToConvert: string) => {
+  return Buffer.from(stringToConvert).toString('base64');
+}
+
+/**
+ * 
+ * @param accountSid 
+ * @param authToken 
+ * @returns - base64 User Credentials
+ */
+export const getBase64Credentials = (accountSid: string, authToken: string): string => {
+  return convertToBase64(accountSid + ":" + authToken);
+}
+
+/**
+ * Resource SID formats
+ * 
+ * Configurations = FJxxxxxxxxxxx;
+ * Versions = FVxxxxxxxxxx;
+ * Plugins = FPxxxxxxxxxxx;
+ * Releases = FKxxxxxxxxx;
+ */
+
+
+export const getAllPlugins = async (context: ApplicationContext) => {
+  const accountSid = await getParam(context, ACCOUNT_SID);
+  const authToken = await getParam(context, AUTH_TOKEN);
+  const base64Credentials = convertToBase64(accountSid + ":" + authToken);
+  const plugins = await axios.get("https://flex-api.twilio.com/v1/PluginService/Plugins?PageSize=20", {
```

Comment:
`axios` has a `params` field in options (the second parameter for `.get`), use that instead of defining your `PageSize` query param as a string. much easier to read an object of query params than a long string